examples/wmt/train.py

```diff
@@ -70,22 +70,30 @@
     'vocab_path', default=None,
     help='Path to load or store sentencepiece vocab file.')
 
+flags.DEFINE_integer(
+    'vocab_size', default=32000,
+    help='Vocabulary size if `vocab_path` is not given.')
+
 flags.DEFINE_string(
-    'dataset_name', default='wmt17_translate/de-en',
+    'dataset_name', default=None,
```

Comment:
Could we preserve the original default tfds datasets for this and the eval dataset?

Reply:
Done. I changed the default value of `reverse_translation` to `False` though to avoid confusion. In the example calls and instructions, this value is set to `True` to maintain the direction of the original example.